landing_page/spec/models/user_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+    let (:user){User.new}
```

Comment:
Why do you defined this variable only to re-define it in the context below? This is unnecessary and can be removed.

Reply:
Resolved in commit 2b36cd84305e63b4adadc6c33091e49dfe3589ef